### PR TITLE
Semi-automated browser testing

### PR DESCRIPTION
**Pull request by eaceaser**
This is semi-automated in the sense that the developer has to run the tests on each target environment. We can look at a cloud CI environment at a later date.

Supported browsers:
* Chrome latest
* Firefox latest
* IE 11 (the latest stable release)

### CODE REVIEW

File: test/util.js [hunk 1]
```diff
@@ -77,22 +77,18 @@ function client() {
   return _client;
 }
 
+function randomString() {
+  return (Math.random() * 0xFFFFFF << 0).toString(16);
```
Comment (erickpintor) [inline test/util.js]: o.O

**Comment by erickpintor** [comment]
🐑  it

File: test/util.js (hunk 1 above)
Comment (eaceaser) [inline test/util.js]: isnt that pro

Comment (eaceaser) [inline test/util.js]: you can thank stackoverflow for this one

Comment (erickpintor) [inline test/util.js]: I won't thank anyone. My brain hurts! :P

**Pull request closed without merging**